web/modules/custom/mymodule/src/Form/SimpleForm.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Locale\CountryManager as CountryManager;
+use Drupal\Core\Messenger\MessengerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Simpleform class using simple form api.
+ */
+class SimpleForm extends FormBase {
+
+  /**
+   * Messenger variable.
+   *
+   * @var object
+   */
+  protected $messenger;
+
+  /**
+   * Call the messenger using container interface for DI.
+   *
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *   Call the container Interface.
+   *
+   * @return object
+   *   It return the messenger functions.
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('messenger')
+    );
+
+  }
+
+  /**
+   * Construct function to call messenger.
+   *
+   * @param \Drupal\Core\Messenger\MessengerInterface $messenger
+   *   Call the messenger interface.
+   */
+  public function __construct(MessengerInterface $messenger) {
+    $this->messenger = $messenger;
+  }
+
+  /**
+   * Get the formid function.
+   *
+   * @return string
+   *   It return the formid.
+   */
+  public function getFormId() {
```

Comment:
@anitha-innoraft - This is a method whose documentation is already present in `ConfigFormBase` class. We don't need to re-write the documentation. You can use https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#inheritdoc
```
/**
 * {@inheritdoc}
 */
```

Reply:
Done